### PR TITLE
Adding openmm-ml

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mikemhenry** [comment]
Using my fork until upstream makes a tag/release

**Comment by mikemhenry** [comment]
I am willing to maintain this package

File: recipes/openmm-ml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - openmm >=7.5
+    - python >=3.6
+
+test:
+  imports:
+    - openmmml
```
Comment (ngam) [inline recipes/openmm-ml/meta.yaml:28]: 😅

File: recipes/openmm-ml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/simplejson/simplejson
```
Comment (ngam) [inline recipes/openmm-ml/meta.yaml:35]:
```suggestion
  home: https://github.com/openmm/openmm-ml
```

**Review by ngam — COMMENTED** [review]
Looking good; you should ping the right team to review/merge.

File: recipes/openmm-ml/meta.yaml (hunk 1 above)
Comment (mikemhenry) [reply recipes/openmm-ml/meta.yaml:28]: Yes very fun :upside_down_face:

File: recipes/openmm-ml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - openmm >=7.5
```
Comment (peastman) [inline recipes/openmm-ml/meta.yaml:23]: This won't work with OpenMM 7.5.  In fact, it won't even really work with 7.7.  To get things fully functional, we need [these fixes](https://github.com/openmm/openmm/milestone/6?closed=1) that will be included in the next release.

File: recipes/openmm-ml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
```
Comment (peastman) [inline recipes/openmm-ml/meta.yaml:21]: I think we also need PyTorch and torchani as dependencies.  Once https://github.com/openmm/openmm-ml/pull/21 is merged, we'll need NNPOps too.

File: recipes/openmm-ml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{% set name = "openmm-ml" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-ml/archive/28b4fadd761221fbe085570faac3e95a2d0be4e0.tar.gz
+  sha256: 913c48641c5fdcbdc6d7ccf14cd33a20ad432c781455e331da86a7a8cc455330
```
Comment (peastman) [inline recipes/openmm-ml/meta.yaml:10]: I think it's cleaner to use `git_url` and `git_rev`.  Here is [an example](https://github.com/conda-forge/openmm-feedstock/blob/18d41142a757522eb291614d937c46a67be084c7/recipe/meta.yaml#L12-L13).  `git_rev` can be either the tag for a release or the hash for revision.

Comment (mikemhenry) [reply recipes/openmm-ml/meta.yaml:10]: I agree, and this is something that really isn't clear in the docs:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos
So I default to using a tarball, but clearly there are examples (like the one you linked) so I will change it and see what they say when they review it, thanks!

File: recipes/openmm-ml/meta.yaml (hunk 3 above)
Comment (mikemhenry) [reply recipes/openmm-ml/meta.yaml:23]: So I know it isn't out yet, but we will need `openmm >=7.7.1` then?

File: recipes/openmm-ml/meta.yaml (hunk 4 above)
Comment (peastman) [reply recipes/openmm-ml/meta.yaml:21]: And also OpenMM-Torch.

**Pull request closed without merging**